Code/Engine/Core/World/Declarations.h

```diff
@@ -143,6 +143,25 @@ struct ezComponentHandle
   friend class ezComponent;
 };
 
+template <typename TYPE>
+struct ezTypedComponentHandle : public ezComponentHandle
```

Comment:
Some comment would be nice, explaining what the advantage / disadvantage over an ezComponentHandle is and when you would prefer one or the other.

Reply:
Added a comment explaining the usage in fce4414